guide/case-studies/saving-satoshi.md

```diff
@@ -0,0 +1,451 @@
+
+**Movie trailer**
+
+On the design and marketing front, a big dream is to do a movie trailer (here’s an [early exploration](https://www.youtube.com/watch?v=EW6tmJMNc5g) from 2023) and poster. When the project first began, the AI image and video generation tools were nowhere near where they had to be to support something like this. A lot has changed in the last two years! Even just over the last few months, the design team has made some very impressive short clips to demonstrate what the technology today is capable of. It’s only going to get better and as it does, the possibility of a movie trailer becomes more likely.
+
+**Community involvement**
+
+In terms of community, the team would like to put more effort towards tapping into the larger bitcoin and software engineering ecosystems to share the game and partner with neighboring groups. More educator tools like a guide for individuals that want to run Saving Satoshi workshops or a comprehensive list of topics that the game covers would help with this initiative.
+
+**Puzzles and interactive content**
+
+The team would like to continue exploring ways to make learning bitcoin tech fun, specifically through more interactive, puzzle-like exercises. This has been one of the project’s goals from the start and it’s one where there is always room for more ideas. The hope is now that version 1 of the game is out, there is bandwidth to iterate on some chapters and find additional ways to creatively engage with the player.
+
+## Conclusion
+
+By design, Saving Satoshi is reflective of the best parts of bitcoin. It has endless rabbit holes. While a cultural transmission in itself, it also brings forth a rich and vibrant world of its own. It’s permissionless and makes players think in ways that they may not be used to. Like bitcoin, it’s 100% open source and built by a passionate group of contributors under the highest standards. It’s by the people, for the people.
+
+It’s a labor of love.
+
+We encourage anyone interested in learning more to play the game and join the community building Saving Satoshi. To get in touch, follow [Saving Satoshi on GitHub](https://github.com/saving-satoshi/saving-satoshi) and join the [#saving-satoshi channel](https://discord.gg/B9WURkwF) on the Bitcoin Design Discord.
+
+Now be quick! [Satoshi needs you](https://savingsatoshi.com).
+
+### Acknowledgements
+
+Saving Satoshi would not be possible without the many people that have come together over the last few years to build it. The core team is made up of Adam Jonas, Christoph Ono (design lead), Satsie, BenAllenG (engineering lead), Matthew Zipkin (lesson content lead), Tuedon Tuoyo, Art Assoiants (product manager), and 4 mysterious cats that Spiral let us borrow to write the story.
```

Comment:
```suggestion
Saving Satoshi would not be possible without the many people that have come together over the last few years to build it. The core team is made up of [Adam Jonas](https://adamjonas.com/), [Christoph Ono](https://www.germanysbestkeptsecret.com/) (design lead), [Satsie](https://satsie.dev), [BenAllenG](https://github.com/benalleng) (engineering lead), [Matthew Zipkin](https://github.com/pinheadmz) (lesson content lead), [Tuedon Tuoyo](https://x.com/TuedonTuoyo), [Art Assoiants](https://www.artassoiants.com/) (product manager), and 4 mysterious cats that [Spiral](https://spiral.xyz) let us borrow to write the story.
```

Reply:
This is updated in my latest commit (did not see your comment here until it was too late).